CONTRIBUTING.md

```diff
@@ -148,6 +150,26 @@ dbg.tcpListen("localhost", 9966)
 5. Start Path of Building Community
 6. Attach the debugger
 
+#### Excluding directories from emmyLua
+
+Depending on the amount of system ram you have avilable and the amout that gets assinged to the jvm running the emmylua language server you might run into issues when trying to debug Path of building.
```

Comment:
Depending on the amount of system ram you have available and the amount that gets assigned to the jvm running the emmylua language server you might run into issues when trying to debug Path of building.